commit 4128dd9afb986321edf0baa8a3aaa3c7c5aa3730
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Oct 7 16:41:32 2010 -0700

    Staging: brcm80211: remove forgotten undef
    
    I forgot to remove the float undefs in typedef.h from previous patches,
    this fixes this issue.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -90,9 +90,6 @@ typedef signed int int32;
 #undef TYPEDEF_UINTPTR
 #undef TYPEDEF_INT16
 #undef TYPEDEF_INT32
-#undef TYPEDEF_FLOAT32
-#undef TYPEDEF_FLOAT64
-#undef TYPEDEF_FLOAT_T
 
 /*
  * Including the bcmdefs.h here, to make sure everyone including typedefs.h
